src/pages/Signup.tsx

```diff
@@ -42,10 +43,12 @@ const Signup: React.FC = () => {
                 email: email
             }
 
-            setSuccess(true);
-            const response = await signupUser(userSigningUp)
 
+            const response = await signupUser(userSigningUp)
+            console.log(response)
```

Comment:
nit: don't leave debugging console logs in production code. It's fine for now though.